src/project/internal/projectconfiguration.cpp

```diff
@@ -285,6 +285,9 @@ io::path_t ProjectConfiguration::defaultSavingFilePath(INotationProjectPtr proje
             }
 
             filename = io::filename(projectPath, false);
+            if (filename == NotationProject::scoreDefaultTitle()) {
```

Comment:
To be honest, I'm not a great fan of comparing strings that may have been translated (we've seen this causing trouble a few times in the past). But in this case I don't see a much better solution, and it seems harmless enough. 

But I wondered one thing: shouldn't we look at the title specified in score properties first, before we resort to looking in the notation itself?